sample-service/src/main/java/uk/ac/ox/ndph/mts/sample_service/security/README.md

```diff
@@ -0,0 +1,111 @@
+# Security
```

Comment:
Should this be in the sample service?
If yes, maybe not under a subfolder but in the root of the service and call it security.md?

Reply:
No it shouldn't, now when merging with main I will move it to the security package